apps/wallet-mobile/src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -471,5 +470,10 @@ const useStyles = () => {
     },
   })
 
-  return styles
+  const colors = {
```

Comment:
are these colors in use?

Reply:
@stackchain  yes, we have in some cases hardcoded colors in the component itself as 
```
color: active === tab ? colors.activeTab : colors.inactiveTab
```